docs/network protocol.md

```diff
@@ -191,6 +191,21 @@ Sent to clients after a client requested this message be sent to them, more info
 ### InvalidPacket
 Sent to clients if the server caught a problem with a packet. This only occurs for errors that are explicitly checked for.
 
+#### Arguments
+| Name | Type | Notes |
+| ---- | ---- | ----- |
+| type | str | The [PacketProblemType](#PacketProblemType) that was detected in the packet |
+| original_cmd | Optional[str] | the `cmd` argument of the faulty packet, will be `None` if the `cmd` failed to be parsed |
+| text | str | A descriptive message of the problem at hand |
+
+##### PacketProblemType
+PacketProblemType indicates the type problem that was detected in the faulty packet, the known problem types are below but others may be added in the future.
+
+| Type | Notes |
+| ---- | ----- |
+| cmd | signaling that the `cmd` argument of the faulty packet could not be parsed correctly |
+| arguments | signaling that the arguments of the faulty packet are not correct |
```

Comment:
If there is a reason to not make this change, let me know and i will approve as is
```suggestion
| cmd | `cmd` argument of the faulty packet could not be parsed correctly |
| arguments | arguments of the faulty packet are not correct |
```